packages/secure-store-utils/android/src/main/java/expo/modules/securestoreutils/SecureStoreUtils.kt

```diff
@@ -0,0 +1,21 @@
+package expo.modules.securestoreutils
+
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+
+class SecureStoreUtils : Module() {
```

Comment:
Do you think we should write a separate issue for the Android implementation piece?

Reply:
Sorry, I guess the implied question is, are you planning on doing the Kotlin work in this PR? If not I'll create an issue to capture the Android side.